recipes/sdl/all/conanfile.py

```diff
@@ -41,6 +41,7 @@ class SDLConan(ConanFile):
         "opengl": [True, False],
         "opengles": [True, False],
         "vulkan": [True, False],
+        "libunwind": [True, False],
```

Comment:
I don't follow. The issue you linked sounds like it completely failed to build without libunwind, but adding it as an option means that it is optional. Could you clarify that a bit further please?

Reply:
Huh, I tried building this fork with both `-o sdl:libunwind=True` and  `-o sdl:libunwind=False` and both worked. I have no idea what's going on here.

Reply:
@Croydon Looks like https://github.com/bincrafters/community/issues/1373

Reply:
libunwind is indeed optional, but without an explicit option.

steps:
- sdl checks for `libunwind.h` availability and define HAVE_UNWIND_H if found: https://github.com/libsdl-org/SDL/blob/25f9ed87ff6947d9576fc9d79dee0784e638ac58/CMakeLists.txt#L820-L826
- then if found, it checks again lubunwind with pkgconfig and I guess that if can fail on some linux distros if pkgconfig file is not installed: https://github.com/libsdl-org/SDL/blob/25f9ed87ff6947d9576fc9d79dee0784e638ac58/CMakeLists.txt#L1319-L1324.

So proposed modification LGTM

Reply:
When building on an archlinux system SDL2 appears to fail to generate cmake without the libunwind package.
This is the partial log within the cmake:
```
-- Checking for one of the modules 'libunwind'
CMake Error at /usr/share/cmake-3.21/Modules/FindPkgConfig.cmake:849 (message):
  None of the required 'libunwind' found
Call Stack (most recent call first):
  source_subfolder/CMakeLists.txt:1321 (pkg_search_module)


-- Checking for one of the modules 'libunwind-generic'
CMake Error at /usr/share/cmake-3.21/Modules/FindPkgConfig.cmake:849 (message):
  None of the required 'libunwind-generic' found
Call Stack (most recent call first):
  source_subfolder/CMakeLists.txt:1322 (pkg_search_module)
```

Later on it explains the arch
```
-- SDL2 was configured with the following options:
-- 
-- Platform: Linux-5.13.12-arch1-1
-- 64-bit:   TRUE
-- Compiler: /usr/bin/cc
-- 
```
#7107 Had the same issues it seems